{"topicSections":[{"title":"Operators","identifiers":["doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/NIOHTTP2Errors\/IOOnClosedConnection\/==(_:_:)"]},{"identifiers":["doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/NIOHTTP2Errors\/IOOnClosedConnection\/init()"],"title":"Initializers"},{"identifiers":["doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/NIOHTTP2Errors\/IOOnClosedConnection\/location"],"title":"Instance Properties"},{"identifiers":["doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/NIOHTTP2Errors\/IOOnClosedConnection\/Equatable-Implementations"],"title":"Default Implementations","generated":true}],"schemaVersion":{"patch":0,"major":0,"minor":3},"hierarchy":{"paths":[["doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2","doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/NIOHTTP2Errors"]]},"sections":[],"abstract":[{"text":"An attempt to perform I\/O was made on a connection that is already closed.","type":"text"}],"identifier":{"url":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/NIOHTTP2Errors\/IOOnClosedConnection","interfaceLanguage":"swift"},"metadata":{"title":"NIOHTTP2Errors.IOOnClosedConnection","externalID":"s:8NIOHTTP20A6ErrorsO20IOOnClosedConnectionV","modules":[{"name":"NIOHTTP2"}],"roleHeading":"Structure","role":"symbol","symbolKind":"struct","fragments":[{"text":"struct","kind":"keyword"},{"text":" ","kind":"text"},{"text":"IOOnClosedConnection","kind":"identifier"}],"navigatorTitle":[{"text":"IOOnClosedConnection","kind":"identifier"}]},"primaryContentSections":[{"kind":"declarations","declarations":[{"platforms":["Linux"],"tokens":[{"text":"struct","kind":"keyword"},{"text":" ","kind":"text"},{"text":"IOOnClosedConnection","kind":"identifier"}],"languages":["swift"]}]}],"variants":[{"paths":["\/documentation\/niohttp2\/niohttp2errors\/ioonclosedconnection"],"traits":[{"interfaceLanguage":"swift"}]}],"relationshipsSections":[{"identifiers":["doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/NIOHTTP2Error","doc:\/\/co.pointfree.VaporRouting\/SQ","doc:\/\/co.pointfree.VaporRouting\/s5ErrorP","doc:\/\/co.pointfree.VaporRouting\/s8SendableP"],"type":"conformsTo","title":"Conforms To","kind":"relationships"}],"kind":"symbol","references":{"doc://co.pointfree.VaporRouting/documentation/NIOHTTP2/NIOHTTP2Errors/IOOnClosedConnection/Equatable-Implementations":{"type":"topic","title":"Equatable Implementations","role":"collectionGroup","abstract":[],"identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/NIOHTTP2Errors\/IOOnClosedConnection\/Equatable-Implementations","url":"\/documentation\/niohttp2\/niohttp2errors\/ioonclosedconnection\/equatable-implementations","kind":"article"},"doc://co.pointfree.VaporRouting/documentation/NIOHTTP2":{"title":"NIOHTTP2","type":"topic","identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2","abstract":[],"role":"collection","kind":"symbol","url":"\/documentation\/niohttp2"},"doc://co.pointfree.VaporRouting/documentation/NIOHTTP2/NIOHTTP2Errors":{"abstract":[{"type":"text","text":"Errors that "},{"type":"reference","identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2","isActive":true},{"type":"text","text":" raises when handling HTTP\/2 connections."}],"title":"NIOHTTP2Errors","kind":"symbol","identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/NIOHTTP2Errors","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"NIOHTTP2Errors"}],"type":"topic","url":"\/documentation\/niohttp2\/niohttp2errors","role":"symbol","navigatorTitle":[{"kind":"identifier","text":"NIOHTTP2Errors"}]},"doc://co.pointfree.VaporRouting/documentation/NIOHTTP2/NIOHTTP2Errors/IOOnClosedConnection/==(_:_:)":{"type":"topic","title":"==(_:_:)","role":"symbol","abstract":[],"identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/NIOHTTP2Errors\/IOOnClosedConnection\/==(_:_:)","fragments":[{"text":"static","kind":"keyword"},{"kind":"text","text":" "},{"kind":"keyword","text":"func"},{"kind":"text","text":" "},{"kind":"identifier","text":"=="},{"kind":"text","text":" "},{"kind":"text","text":"("},{"preciseIdentifier":"s:8NIOHTTP20A6ErrorsO","kind":"typeIdentifier","text":"NIOHTTP2Errors"},{"kind":"text","text":"."},{"preciseIdentifier":"s:8NIOHTTP20A6ErrorsO20IOOnClosedConnectionV","kind":"typeIdentifier","text":"IOOnClosedConnection"},{"kind":"text","text":", "},{"preciseIdentifier":"s:8NIOHTTP20A6ErrorsO","kind":"typeIdentifier","text":"NIOHTTP2Errors"},{"kind":"text","text":"."},{"preciseIdentifier":"s:8NIOHTTP20A6ErrorsO20IOOnClosedConnectionV","kind":"typeIdentifier","text":"IOOnClosedConnection"},{"kind":"text","text":") -> "},{"preciseIdentifier":"s:Sb","kind":"typeIdentifier","text":"Bool"}],"url":"\/documentation\/niohttp2\/niohttp2errors\/ioonclosedconnection\/==(_:_:)","kind":"symbol"},"doc://co.pointfree.VaporRouting/documentation/NIOHTTP2/NIOHTTP2Errors/IOOnClosedConnection/init()":{"url":"\/documentation\/niohttp2\/niohttp2errors\/ioonclosedconnection\/init()","role":"symbol","identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/NIOHTTP2Errors\/IOOnClosedConnection\/init()","abstract":[],"fragments":[{"kind":"identifier","text":"init"},{"kind":"text","text":"()"}],"type":"topic","kind":"symbol","deprecated":true,"title":"init()"},"doc://co.pointfree.VaporRouting/s5ErrorP":{"title":"Swift.Error","type":"unresolvable","identifier":"doc:\/\/co.pointfree.VaporRouting\/s5ErrorP"},"doc://co.pointfree.VaporRouting/documentation/NIOHTTP2/NIOHTTP2Errors/IOOnClosedConnection":{"url":"\/documentation\/niohttp2\/niohttp2errors\/ioonclosedconnection","navigatorTitle":[{"kind":"identifier","text":"IOOnClosedConnection"}],"abstract":[{"type":"text","text":"An attempt to perform I\/O was made on a connection that is already closed."}],"role":"symbol","kind":"symbol","fragments":[{"kind":"keyword","text":"struct"},{"kind":"text","text":" "},{"kind":"identifier","text":"IOOnClosedConnection"}],"type":"topic","title":"NIOHTTP2Errors.IOOnClosedConnection","identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/NIOHTTP2Errors\/IOOnClosedConnection"},"doc://co.pointfree.VaporRouting/documentation/NIOHTTP2/NIOHTTP2Errors/IOOnClosedConnection/location":{"role":"symbol","fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"location","kind":"identifier"},{"text":": ","kind":"text"},{"text":"String","preciseIdentifier":"s:SS","kind":"typeIdentifier"}],"url":"\/documentation\/niohttp2\/niohttp2errors\/ioonclosedconnection\/location","title":"location","type":"topic","kind":"symbol","identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/NIOHTTP2Errors\/IOOnClosedConnection\/location","abstract":[{"text":"The location where the error was thrown.","type":"text"}]},"doc://co.pointfree.VaporRouting/documentation/NIOHTTP2/NIOHTTP2Error":{"abstract":[{"text":"The base protocol for all errors thrown by ","type":"text"},{"isActive":true,"identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2","type":"reference"},{"text":".","type":"text"}],"url":"\/documentation\/niohttp2\/niohttp2error","title":"NIOHTTP2Error","role":"symbol","fragments":[{"kind":"keyword","text":"protocol"},{"kind":"text","text":" "},{"kind":"identifier","text":"NIOHTTP2Error"}],"type":"topic","identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/NIOHTTP2Error","kind":"symbol","navigatorTitle":[{"kind":"identifier","text":"NIOHTTP2Error"}]},"doc://co.pointfree.VaporRouting/SQ":{"type":"unresolvable","title":"Swift.Equatable","identifier":"doc:\/\/co.pointfree.VaporRouting\/SQ"},"doc://co.pointfree.VaporRouting/s8SendableP":{"type":"unresolvable","title":"Swift.Sendable","identifier":"doc:\/\/co.pointfree.VaporRouting\/s8SendableP"}}}